pkg/controller/remotewatcher/remotewatcher_controller.go

```diff
@@ -173,6 +173,26 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Indexes
+	indexer := mgr.GetFieldIndexer()
+
+	// Pod
+	err = indexer.IndexField(
```

Comment:
I _think_ this is needed here because field indexes appear to be stored on the cache within each manager, and since I'm re-introducing remote managers with this PR we have new need of this index.